src/Uno.UI/UI/Xaml/Controls/ContentControl/ContentControl.skia.cs

```diff
@@ -25,6 +25,16 @@ public partial class ContentControl
 			RemoveChild(ContentTemplateRoot);
 		}
 
-		protected override Size MeasureOverride(Size availableSize) => base.MeasureOverride(availableSize);
+		protected override Size MeasureOverride(Size availableSize)
+		{
+			// This is the same as Control.MeasureOverride, except that we use Content if FindFirstChild is null.
```

Comment:
Can you create an issue to track this issue specifically? Just so we can include it in lifecycle fixes backlog

Reply:
@MartinZikmund I guess this is already https://github.com/unoplatform/uno/issues/190 ?

The issue mentions:

> On Windows the template of a control is being applied only when this control is in the visual tree and either we explicitly invoke the ApplyTemplate method, or we wait for the first measure pass.

The case I'm dealing with here is the "wait for the first measure pass". I'll add a link to the issue in code comments.